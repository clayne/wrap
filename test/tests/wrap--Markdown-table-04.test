wrap | /dev/null | -u | md-table-04.md | 0
