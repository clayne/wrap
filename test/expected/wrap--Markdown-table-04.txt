This is a line of text that just so happens to have a | in it.
This is a second line of text.  This is a third line of text.  This is a fourth
line of text.
